src/consumer/mod.rs

```diff
@@ -178,21 +180,18 @@ impl<F> ConsumerBuilder<F> {
         self.workers = w;
         self
     }
-}
 
-impl<F, E> ConsumerBuilder<F>
-where
-    F: Fn(Job) -> Result<(), E> + Send + Sync + 'static,
-{
     /// Register a handler function for the given job type (`kind`).
     ///
     /// Whenever a job whose type matches `kind` is fetched from the Faktory, the given handler
     /// function is called with that job as its argument.
-    pub fn register<K>(&mut self, kind: K, handler: F) -> &mut Self
+    pub fn register<K, J>(&mut self, kind: K, handler: J) -> &mut Self
```

Comment:
Can we make this `H` for `Handler`?

Reply:
Sure; it was `J` for `JobHandler`.  :man_facepalming: